docs/service-catalog/03-01-provisioning.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Provisioning and binding flow
+type: Details
+---
+
+The diagram shows an overview of interactions between all resources related to the processes of provisioning and binding:
+
+![Provisioning and binding](./assets/provisioning-and-binding.svg)
+
+## Provision a service
+
+Provisioning a service means creating an instance of a service. To provision a service, create a ServiceInstance custom resource. During this process, the Service Broker creates a new instance of a service. The form and scope of this instance depends on the Service Broker.
+
+![Provisioning](./assets/provisioning.svg)
+
+## Bind a service
+
+Binding is a process of connecting a ServiceInstance and an Application. The binding process consists of two steps:
+
+![Binding](./assets/binding.svg)
+
+- Create a ServiceBinding. When you create a ServiceBinding, the Service Catalog creates a Secret with credentials needed to access the Application.
+- Create a ServiceBindingUsage. After creating this custom resource, ServiceBindingUsage uses the [PodPreset](https://kubernetes.io/docs/concepts/workloads/pods/podpreset/) to inject credentials into the Application, which allows you to consume the service.
```

Comment:
I'd stay with the imperative mood since I'd like to emphasize that it's the user who must perform these steps. Gerund might sound as if the operation is performed automatically under the hood.